x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Analyzer.java

```diff
@@ -1173,52 +1170,6 @@ protected LogicalPlan rule(OrderBy ob) {
         }
     }
 
-    private static class ImplicitCasting extends AnalyzerRule<LogicalPlan> {
```

Comment:
There's a commented instantiation of this rule ([L124](https://github.com/elastic/elasticsearch/pull/76638/files?file-filters%5B%5D=.java&file-filters%5B%5D=.txt#diff-098bb3ac8796851bf661076f1d4db0ef756794182fd40baaef0ff4d2cc47cfccL124)). Not sure why this was kept, but I think commented/dead code isn't typically kept around, so maybe  remove that line too.

Reply:
👍 I didn't thin about searching for the class. Thanks!